VideoLocker/res/layout/row_course_dashboard_list.xml

```diff
@@ -1,36 +1,36 @@
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    android:id="@+id/chapter_row_layout"
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/chapter_row_layout"
     android:layout_width="match_parent"
-    android:layout_height="@dimen/dashboard_navigation_element_height"
+    android:layout_height="@dimen/dashboard_nav_element_height"
```

Comment:
Bounding the description to one line is probably not the right thing to do though. It's probably not an issue practically at the moment, as the descriptions are quite short, but with translations they might grow larger. Anyway, I guess it's fine for now.